alethzero/MainWin.cpp

```diff
@@ -860,8 +860,8 @@ void Main::readSettings(bool _skipGeometry)
 		p->readSettings(s);
 	});
 
-	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setAsk(u256(s.value("askPrice", "500000000000").toString().toStdString()));
-	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setBid(u256(s.value("bidPrice", "500000000000").toString().toStdString()));
+	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setAsk(u256(s.value("askPrice", "50000000000").toString().toStdString()));
+	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setBid(u256(s.value("bidPrice", "50000000000").toString().toStdString()));
```

Comment:
so I need to convert u256 to std string and the to QString, right? No short cuts? since we are suing u256 a lot.